### DISCUSSION TITLE
检测到当前网络为”按流星计费的网络“时，将视频切换至较低的画质

### DISCUSSION THREAD

**Discussion by DeepChirp** [the1812/Bilibili-Evolved#4478] [功能建议 / Ideas]
能否推出一个功能，当检测到当前网络为”按流星计费的网络“时，将视频切换至较低的画质，以便节省流量呢？谢谢

**Comment by the1812** [comment]
检测是能检测, 但是没有切画质的功能

**Reply by WakelessSloth56** [reply]
切换画质的播放器接口：https://github.com/the1812/Bilibili-Evolved/discussions/4341#discussioncomment-7499077

**Reply by the1812** [reply]
nb, 那可以搞一搞了
